### ISSUE TITLE
Connection test failed: NoHttpResponseException (AWS OpenSearch 2.5)

### ISSUE THREAD

**Issue by lars85** [AntonShuvaev/elasticsearch4idea#103]
I am not able to connect to AWS OpenSearch Service with jumping host.

Is there a way to provide you more information?

#### General-Tab

- Elasticsearch-URL: `https://vpc-spryker-elasticsearch-016-wv76pe4p4vzidpxazovmpxq73i.eu-central-1.es.amazonaws.com`

### SSH

- Use SSH tunnel: yes
- Host: test.de
- Test SSH Connection: `Connection test successfull`

### Test Connection

```
Connection test failed: org.apache.http.NoHttpResponseException: 127.0.0.1:59763 failed to respond
```

--------

# Shell test

```shell
ssh test.de
curl https://vpc-spryker-elasticsearch-015-a3t56g4pphuge2yyqu4m4o2f2q.eu-central-1.es.amazonaws.com
```

```json
{
  "name" : "394cb4a3ee8835c746146388812b415a",
  "cluster_name" : "812064093403:spryker-elasticsearch-015",
  "cluster_uuid" : "oMBGr2LzQeqnuOTbcpjeAA",
  "version" : {
    "distribution" : "opensearch",
    "number" : "2.5.0",
    "build_type" : "tar",
    "build_hash" : "unknown",
    "build_date" : "2023-03-02T04:30:45.421313Z",
    "build_snapshot" : false,
    "lucene_version" : "9.4.2",
    "minimum_wire_compatibility_version" : "7.10.0",
    "minimum_index_compatibility_version" : "7.0.0"
  },
  "tagline" : "The OpenSearch Project: https://opensearch.org/"
}
```

**Comment by AntonShuvaev** [comment]
Not all options of the SSH config file are supported. Can you share ssh config file without sensitive data?

**Comment by AntonShuvaev** [comment]
Fixed in 2023.3.3
